server.js

```diff
@@ -290,13 +290,13 @@ async function getSavedGamesById(server) {
 async function postResult(server) {
   const { username, won, lost, draw } = await server.body;
   let finalScore = await calculateScore(won, lost, draw);
-  let user_id = [
-    ...(await client.queryObject({
```

Comment:
Issue with spread operator in postgre. Removing it fixes typeError with post request